base/BUILD

```diff
@@ -1,11 +1,10 @@
-package(default_visibility = ["//visibility:public"])
-
 load(":base.bzl", "NONROOT", "distro_components")
 load(":distro.bzl", "DISTROS")
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
```

Comment:
should this use: load("@rules_pkg//:pkg.bzl", "pkg_tar") ?

Reply:
thanks. fixed it!